app/helpers/rails_admin/application_helper.rb

```diff
@@ -31,7 +31,7 @@ def edit_user_link
 
       if authorized
         edit_url = rails_admin.url_for(
-          action_name: edit_action.action_name,
```

Comment:
It looks like this was originally `action`, and changed to `action_name` in https://github.com/railsadminteam/rails_admin/pull/3475 - @thedanielhanke was that needed for Pundit? Or maybe made in error?

Reply:
I believe it was by mistake, and I overlooked it too.